*** tests/A-01-02-M-ok.out	2022-09-30 09:03:54.000000000 +0100
--- tests/A-01-02-M-ok.outhyp	2022-10-21 20:36:44.063317881 +0100
***************
*** 31,42 ****
  5 - Menu de consultas
  6 - Ver saldo global
  0 - Sair
- Escolha uma opção: Ficheiro sem nome. Guardar ficheiro como: MENU PRINCIPAL
- 1 - Abrir
- 2 - Guardar
- 3 - Gestão de clientes
- 4 - Gestão de terminais
- 5 - Menu de consultas
- 6 - Ver saldo global
- 0 - Sair
  Escolha uma opção: 
\ No newline at end of file
--- 31,34 ----
